packages/paste-core/components/button/src/styles.ts

```diff
@@ -8,9 +8,9 @@ import {ButtonWrapperProps, ButtonChildrenProps} from './types';
 /*
  * Sizes
  */
-const sizeReset = css`
+const sizeReset = (props: ButtonWrapperProps): SerializedStyles => css`
   font-size: 100%;
-  padding: 0;
+  padding: ${themeGet('space.space10')(props)};
```

Comment:
This should be a hard 0 too, because no matter what theme is provided we want it to remain 0.